Readme.md

```diff
@@ -266,7 +266,7 @@ TIPS
    - Builds a HTML report from JSON with support for debug msgs & embedded Base64 images.
 
 ### General
- - [SQL schema format] use :ruby schema format to get faster parallel:prepare`
+ - [SQL schema format] use [the `:sql` schema format](https://guides.rubyonrails.org/active_record_migrations.html#types-of-schema-dumps) to get faster parallel:prepare`
```

Comment:
did you benchmark that ?

Reply:
original came from https://github.com/grosser/parallel_tests/commit/fa1e0f9d3a572f7aa67fc506b1889fdf96c9a989 🤷‍♂

Reply:
`# there is no separate dump / load for schema_format :sql -> do it safe and slow` <- so that's no longer true ?

Reply:
... yeah old code is still there so idk ...

Reply:
Rails has `db:structure:dump` and `db:structure:load` tasks that create/read from `db/structure.sql`, and I'm pretty sure these are faster than the `db:schema` tasks so I thought that was just a typo in the README, since the heading was `[SQL schema format]`. But I will run a benchmark to test!

Reply:
the tasks.rb needs to actually use the `db:structure:dump` then too ... that did not exist before and is why the warning was added